src/plugins/orangehrmRecruitmentPlugin/Dao/RecruitmentAttachmentDao.php

```diff
@@ -73,11 +75,25 @@ public function getVacancyAttachmentById(int $attachId): ?VacancyAttachment
         return null;
     }
 
+    /**
+     * @param int $candidateId
+     * @return RecruitmentAttachment|null
+     */
+    public function getCandidateAttachmentByCandidateId(int $candidateId): ?RecruitmentAttachment
```

Comment:
```suggestion
    public function getPartialCandidateAttachmentByCandidateId(int $candidateId): ?RecruitmentAttachment
```
https://www.doctrine-project.org/projects/doctrine-orm/en/2.13/reference/partial-objects.html
This is a deprecated pattern. but we are going to achieve this

Reply:
done